src/Turbo/Resources/assets/src/turbo_controller.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { Controller } from 'stimulus';
+import * as Turbo from '@hotwired/turbo';
+
+// Expose Turbo to the rest of the app to allow for dynamic Turbo calls
+window.Turbo = Turbo;
```

Comment:
Is exposing it as a global variable the correct/normal way to do things? Where is it needed globally?

Reply:
I wasn't sure about this one, but it has two benefits I can see: it helps usage from outside Webpack (ie in a legacy JS static file for instance) and it ensures Webpack subtree split won't remove the dependency at compilation.

I'm not sure it's necessary, it's debatable.

Reply:
I'd be in favor or removing it then, if this plays well with subtree splitting from webpack. How can we check this?

Reply:
We should keep this as it's used by native adapters.